src/components/Home/ClusterDashboardItem.js

```diff
@@ -45,29 +79,27 @@ const WrapperDeleted = styled.div`
   background-color: ${props => props.theme.colors.darkBlueDarker1};
 `;
 
-const LabelWrapper = styled.div`
-  flex: 0 0 90px;
-  font-size: 1.2em;
-`;
-
-const ContentWrapper = styled.div`
-  flex: 1;
-  padding-right: 15px;
-`;
-
-const TitleWrapper = styled.div`
-  font-size: 1.2em;
-  z-index: 120;
-  position: relative;
-`;
-
-const NameWrapper = styled.span`
-  font-weight: 700;
-`;
-
 const ButtonsWrapper = styled.div`
-  text-align: right;
   flex: 0 0 210px;
+  display: flex;
+  justify-content: flex-end;
+  ${mq('medium')} {
+    flex: unset;
+    position: absolute;
+    top: 21px;
+    right: 8px;
+  }
+  ${mq('725')} {
```

Comment:
```suggestion
  ${mq(725)} {
```

Reply:
Here I was thinking in types already: `breakpoint` argument is a string, that's why I passed a string.

Reply:
I think it would raise confusion, as someone might think "should this be `'725'` or `'725px'`?".
As for types, the argument type can be an union (`string | number`), and accept both strings and numbers.